<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>100% legal youtube download no virus free</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>flush.png</normaloff>flush.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="pathLabel">
      <property name="text">
       <string>Download Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="PathLine"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="PathButton">
      <property name="text">
       <string>Choose Path</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="URLLabel">
      <property name="text">
       <string>Video URL</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="URLLine">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="ThumbLabel">
      <property name="text">
       <string>Thumbnail preview</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="URLButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Get URL</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="ThumbnailPlaceholder">
      <property name="minimumSize">
       <size>
        <width>128</width>
        <height>72</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../meme stash/clarkson/Capture.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="DownloadButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Download (MP4)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DownloadMP3Button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Download (MP3)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
